Fix DmnXMLConverter exception: cvc-elt.1 (flowable#255)

ouyida3 opened the pull request:
now when I visit http://www.omg.org/spec/DMN/20151101 , get 404 error.
Should I change it into http://www.omg.org/spec/DMN/20151101/dmn.xsd?

I get an exception cvc-elt.1 when I use DMN.
So I change the code:`flowable-engine/modules/flowable-dmn-xml-converter/src/main/resources/org/flowable/impl/dmn/parser/dmn.xsd`, and `mvn install` a new jar.
With this new jar, the exception disappeared.

tijsrademakers commented:
The XSD is the official DMN 1.1 XSD that is provided by the OMG specification website. This XSD can't be changed.

ouyida3 commented:
> The XSD is the official DMN 1.1 XSD that is provided by the OMG specification website. This XSD can't be changed.

But the official DMN 1.1 XSD is different from `flowable-engine/modules/flowable-dmn-xml-converter/src/main/resources/org/flowable/impl/dmn/parser/dmn.xsd`. 
I visit [https://www.omg.org/spec/DMN/1.1](https://www.omg.org/spec/DMN/1.1), and find the XSD:
[https://www.omg.org/spec/DMN/20151101/dmn.xsd](https://www.omg.org/spec/DMN/20151101/dmn.xsd)

You will see `xsd:schema xmlns` is :
`http://www.omg.org/spec/DMN/20151101/dmn.xsd`, not `http://www.omg.org/spec/DMN/20151101`.
So I changed it.
After I changed it, the DMN errer(cvc-elt.1 ) disappeared.

I commented the `cvc-elt.1` Exception details in flowable#255.
https://github.com/flowable/flowable-engine/issues/255

tijsrademakers commented:
You are right, we are looking into why this is different.

yvoswillens commented:
Hi @ouyida3 
I just pushed a change to makes the new final 1.2 DMN spec the default when generating DMN XMLs.
Parsing support was already implemented.
This will make use of the 1.2 xmlns "http://www.omg.org/spec/DMN/20180521/MODEL/" from now on.

Regards,

Yvo

ouyida3 commented:
hi, @yvoswillens ,I just tested it.Test success. Thank you very much :)
At the same time, I found that my "xxx.dmn" file
`<definitions xmlns="http://www.omg.org/spec/DMN/20151101/dmn.xsd" ...` runs successfully today, while it failed the day before yesterday.
Maybe OMG changed something these two days?
Anyway thank u very much!
